### DISCUSSION TITLE
Additional Environmental Sensors - Should there be more?

### DISCUSSION THREAD

**Discussion by Derek-Gutheil** [Ribbit-Network/ribbit-network-frog-hardware#113] [Ideas]
There are other types of local environmental things that can be tracked. I've seen things like methane tossed around as an idea and I wanted to start a chat about other things that could be measured.
- Ambient Light
- Ambient Sound Levels: Can be used to measure noise pollution
- AQI: Air Quality Index. A measurement of particulate matter in the air

Any others?

**Comment by keenanjohnson** [comment]
I love this discussion thread. Thanks so much for starting it @Derek-Gutheil!

I've really wanted to add additional gas sensors like Methane and NO2, but so far we haven't identified a cheap-ish high-quality option.

Additionally, it's been proposed to me previously that a VoC (Volatile Organic Compounds) sensor could be interesting for human health. 

We could also look into a particulate sensor, though I feel like there are already very wide spread particulate networks like PurpleAir, so I'm not sure this would add any value into the world.

**Reply by Derek-Gutheil** [reply]
Ah, I wasn't aware of the PurpleAir network.

Other gasses: What defines a high-quality solution? I assume something like this wouldn't work? https://wiki.seeedstudio.com/Grove-Multichannel-Gas-Sensor-V2/

**Comment by keenanjohnson** [comment]
It's a good question and it's still something I'm trying put a more specific definition on. But in general, we try to look for sensors that provide direct measurement with a relatively high degree of accuracy. For example, our CO2 sensors today (SCD30) use NDIR technology which is of a high quality.

The mems sensors like the SEED ones you linked above, aren't known for being accurate and are best used for questions like "is this gas present or not?" rather than, "How does the percentage of this gas change over time".

**Reply by Aravind-Krishnadeva** [reply]
Hi, Keenan, Got some good insights following this thread. I have been too, investigating some low powered sensors, that can be powered by solar cells. (which I have put up in the discussion forum).

**Comment by Derek-Gutheil** [comment]
Has there been any thought to measuring background radiation? Here is an open source device designed to do that, doesn't look like there are plug and play modules to do it though: https://multigeiger.readthedocs.io/en/latest/about.html

**Reply by Derek-Gutheil** [reply]
There are a few commerical-ish solutions:
- https://www.sparkfun.com/products/retired/11345
- https://www.sparkfun.com/products/14209

**Comment by keenanjohnson** [comment]
Hey @Derek-Gutheil ! We've been mostly focused on climate change type sensors, so I don't think anyone has explored radiation yet.

I know that https://safecast.org/ already does a pretty good job on the radiation side of things, so if we did do radiation, we should consider collaborating there :)
